package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@sentry/browser": "6.1.0",
     "@sentry/core": "6.1.0",
+    "@sentry/hub": "6.1.0",
```

Comment:
There's a package just for the Hub now?

Reply:
Yep, and we need to import it to use `getCurrentHub`